docs/glossary.md

```diff
@@ -0,0 +1,361 @@
+
+
+## I
+
+---
+
+#### Immature ticket
+
+Once tickets are mined they are immature for 256 blocks (about 20 hours) and cannot be called to vote until after this period elapses.
+
+#### Inflation
+
+Increase in the [available supply](https://docs.decred.org/advanced/inflation/) of Decred as new DCR is minted into existence through the block reward.
+
+
+## I
+
+---
+
+#### Live ticket
+
+Tickets that are waiting to be called.
+
+
+## M
+
+---
+
+#### Mainnet
+
+The proper Decred network - a term used to differentiate from testnet.
+
+#### Mempool
+
+Transactions waiting to be mined.
+
+#### Miners
+
+Miners help secure the network via Proof-of-Work (PoW) mining. Miners recieve block rewards and transaction fees for successfully validating blocks. 
+
+#### Mining Pool
+
+A group of miners who share (pool) their computational resources to mine DCR. When a member of a mining pool successfully mines a block, the rewards are typically split among all pool members in proportion to the hashpower they contribute to the pool. 
+
+#### Missed ticket
+
+Tickets that have been called but did not receive a reward. This can happen if a ticket is called to vote and does not respond. This can also happen if a ticket is called to vote, responds and broadcasts their vote to the network, but a miner does not include the vote in the following block. 
+
+## O
+
+---
+
+#### Orphan block
+
+Orphaned blocks are valid blocks which are not included in the definitive blockchain. Orphan blocks can occur when they are part of a branch of the blockchain that has been abandoned. This can occur naturally when two miners produce blocks at similar times. Orphan blocks can also be created when they build on an unknown block (i.e. the "parent" block is unknown, making it an "orphan"). 
+
+
+#### Orphan transaction
+
+A transaction with missing inputs (i.e. the "parent" transaction is unknown, making the transaction an "orphan"). Orphan transactions can also be created when they are part of a block that has been abandoned and have not been included in another block. For example, if a miner creates a valid block with transactions specific to them, such as coinbase transactions and votes, and that block is orphaned, the transactions in that block will become orphaned transactions. 
+
+
+## P
+
+---
+
+#### Politeia
+
+A system for facilitating the submission and discussion of [proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
+
+#### Private key
+
+An astronomically large secret number that allows DCR to be spent. All transactions that spend DRC must include a signature that can only be produced with knowledge of this secret number.
```

Comment:
DRC -> DCR

> All transactions ... must include a signature that can only be produced with knowledge of this secret number.

This is not universally true.  The only requirement is that the script virtual machine is able to execute the scripts in the input and output without failure.  There is no requirement that a signature check is performed, but this is the most common way to protect DCR from being spent by unauthorized parties.  One example of spending requirements that does not perform any signature validation is a puzzle, in which case any arbitrary data can be provided and so long as it hashes to the expected value, the script executes and the output can be spent.

Reply:
Googled around trying to understand this better. Couldn't come up with a wording that keeps the level of detail we have here and is still accurate. Have retreated to a higher level of abstraction (below). If you can think of more useful wording, can use that.

#### Private key

An astronomically large secret number that allows DCR to be spent. All transactions that spend DCR can only be produced with knowledge of this secret number.

Reply:
Would go with:

> An astronomically large number which, when kept secret from others, enables legitimate spending of personal Decred while preventing others from spending it.  Private keys are encrypted by wallets to prevent theft.

Reply:
Implemented suggestion.